### PR TITLE
Add Python Script for Realtime-Face-Detection

### PR DESCRIPTION
**Pull request by pksgmv**
# Description

Python Script for a Realtime Face Detection App using Python-OpenCV.

Fixes #31 

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by AdityaJ7** [comment]
hey @pksgmv can you remove the xml file and just mention a link from which it can be downloaded?

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
hey @pksgmv can you remove the xml file and just mention a link from which it can be downloaded?
also adjust your readme file and add a requirments.txt file

**Comment by pksgmv** [comment]
> hey @pksgmv can you remove the xml file and just mention a link from which it can be downloaded?
> also adjust your readme file and add a requirments.txt file
> 

Okay @AdityaJ7 I'll make the above changes and then make a PR...

**Review by tusharnankani — COMMENTED** [review]
> > hey @pksgmv can you remove the xml file and just mention a link from which it can be downloaded?
> 
> Okay @AdityaJ7 I'll make the above changes and then make a PR...

The `.xml` file is still there, please remove it.

 > also adjust your readme file and add a requirments.txt file

Also, you can refer to the [README Template,](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)

File: Scripts/Miscellaneous/Realtime_Face_Detection/face_detector.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    for (x, y, w, h) in face_coordinates:
+        cv.rectangle(frame, (x,y), (x+w,y+h), (randrange(256),randrange(256),randrange(256)), 5)
+
+    # Display the image with the faces
+    cv.imshow('Face Detector', frame)
+    key = cv.waitKey(1)
+
+    # Stop when the Q key is pressed
+    if key == 81 or key == 113:     # ASCII codes for Q and q
+        break
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/Realtime_Face_Detection/face_detector.py:35]: Fix the DeepSource error.

**Comment by pksgmv** [comment]
@tusharnankani Okay. I'll make another PR looking into the errors...

**Comment by tusharnankani** [comment]
> @tusharnankani Okay. I'll make another PR looking into the errors...

You don't need to create another PR, you can push the changes directly to the respective branch you checked out to, and the changes will be made.

**Review by chavarera — APPROVED** [review]

**Pull request closed without merging**